### DISCUSSION TITLE
Odd Error

### DISCUSSION THREAD

**Discussion by SAFNM** [Open-Shell/Open-Shell-Menu#1152] [Q&A]
Windows 11 Pro 21H2, Open Shell 4.4.175

Windows Task Bar set to left. From time to time, the Open Shell icon moves from the start button location to the middle of the taskbar. Clicking on the Open Shell icon restores it to the start button location at the far left end of the task bar.

**Comment by CTVCAM8** [comment]
This is the same temporary behavior that bonzibudd mentioned in #745
Never seen the start button jump yet.
